Utilities/build-script-helper.py

```diff
@@ -93,7 +91,7 @@ def add_common_args(parser):
     parser.add_argument('--sanitize-all', action='store_true', help='build using every available sanitizer in sub-directories of build path')
     parser.add_argument('--verbose', '-v', action='store_true', help='enable verbose output')
 
-  subparsers = parser.add_subparsers(title='subcommands', dest='action', metavar='action')
+  subparsers = parser.add_subparsers(title='subcommands', dest='action', required=True, metavar='action')
```

Comment:
Does PR CI run on Ubuntu 18.04? 3.7 was released in 2018, so wouldn't surprise me if it didn't have it.

Reply:
I just checked, they run Python 3.6.9 😬 I guarded the `required=True` option behind a Python version check.